site/docs/v1/tech/apis/two-factor.adoc

```diff
@@ -435,6 +435,9 @@ include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation.adoc[
 ==== Request Body
 
 [.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.46.0#::
+The Application Id. Provide this value to use an application-specific email template and make `application` available as a template variable.
```

Comment:
Can you review the usage of this type of parameter on the User API and make some adjustments to be closer to that usage?